app.js

```diff
@@ -0,0 +1,38 @@
+const express = require("express");
+const dotenv = require("dotenv");
+const mongoose = require("mongoose");
+const cors = require("cors");
+const {requireAuth} = require("./middleware/authMiddleware");
+
+dotenv.config();
+const app = express();
+
+// connect to db
+mongoose
```

Comment:
Ideally, we would want to 'await' this and ensure mongoose is connected before our app starts listening for requests